scripts/binary/index.js

```diff
@@ -143,12 +144,82 @@ const deploy = {
         fail('Release Failed')
         throw err
       })
+      .then(() => {
+        return this.checkDownloads({ version })
+      })
     }
 
     return askMissingOptions(['version'])(options)
     .then(release)
   },
 
+  checkDownloads ({ version }) {
+    const systems = [
+      { platform: 'linux', arch: 'x64' },
+      { platform: 'darwin', arch: 'x64' },
+      { platform: 'win32', arch: 'ia32' },
+      { platform: 'win32', arch: 'x64' },
+    ]
+
+    const checkSystem = ({ platform, arch }) => {
+      const url = `https://download.cypress.io/desktop/${version}?platform=${platform}&arch=${arch}`
+      const system = `${platform}-${arch}`
+
+      process.stdout.write(`Checking for ${chalk.yellow(system)} at ${chalk.cyan(url)} ... `)
+
+      return urlExist(url)
```

Comment:
mild nit, why not use request-promise here? it will throw on a non-200 error code

i have a dep vendetta

Reply:
Good point. All it seems to do is the equivalent of `request.head()`, so I can change this to do the same without the extra dep.